design/Status/line_to_line.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="25" height="24" viewBox="0 0 25 24" xmlns="http://www.w3.org/2000/svg">
+<path d="M3.43594 17V7.00586H5.55508V15.25H9.60879V17H3.43594ZM10.6611 14.1016V12.3926H14.332V14.1016H10.6611ZM16.1637 17V7.00586H18.2828V15.25H22.3365V17H16.1637Z" />
```

Comment:
seems like a strange icon. What is the use case behind this?

Reply:
it has width 25

Reply:
This is meant to represent Line-to-Line and Line-to-Neutral which are ways of measuring current, power, voltage. These would be used as icons in a toggle button or option menu, to switch between the two metering methods. They might also be used inline or in row to indicate if a particular measurement uses one of the methods over another.

Fixed the 25 width - I exported these ones from Figma and had a lot of issues...